server/service/osquery_utils/queries.go

```diff
@@ -129,7 +130,17 @@ var hostDetailQueries = map[string]DetailQuery{
 				return nil
 			}
 
-			if rows[0]["major"] != "0" || rows[0]["minor"] != "0" || rows[0]["patch"] != "0" {
+			if rows[0]["name"] == "Ubuntu" {
```

Comment:
nit, following a pattern in this file:

```suggestion
			strings.Contains(strings.ToLower(rows[0]["name"]), "ubuntu") {
```